Makefile

```diff
@@ -50,6 +50,7 @@ BUILDMNT = /go/src/$(GOTARGET)
 BUILD_IMAGE ?= golang:1.13.0-stretch
```

Comment:
Remove stretch just use golang so windows can build binary too

Reply:
Not important right now actually; I dont think that is required to accomplish our goal here.